playground/css/__tests__/css.spec.ts

```diff
@@ -400,7 +400,8 @@ test('inlined-code', async () => {
   }
 })
 
-test('minify css', async () => {
+// TODO: need to convert esbuild target syntax to lightningcss
```

Comment:
You can grab some code from here: https://github.com/ArnaudBarre/downwind/blob/main/src/utils/convertTargets.ts

Reply:
Thanks for the pointer. If the vite maintainers end up switching to lightningcss, I'm sure that would be useful!